addons/stock/models/stock_orderpoint.py

```diff
@@ -31,6 +39,8 @@ def default_get(self, fields):
     name = fields.Char(
         'Name', copy=False, required=True, readonly=True,
         default=lambda self: self.env['ir.sequence'].next_by_code('stock.orderpoint'))
+    trigger = fields.Selection([
+        ('auto', 'Auto'), ('manual', 'Manual')], string='Trigger', default='auto', required=1)
```

Comment:
required=True
also maybe it'd make more sense to just add a manual boolean, since i don't see which new trigger we'd need?